https-upgrades/tests.json

```diff
@@ -0,0 +1,158 @@
+{
+    "navigations": {
+        "name": "Navigations",
+        "desc": "Navigations should be upgraded to https",
+        "tests": [
+            {
+                "name": "bloom filter match - should be upgraded",
+                "siteURL": "http://stackoverflow.com/path/file.html?search=query&another=one#fragment",
```

Comment:
Those are real hostnames because the only test data (bloom filters and friends) that we had were based on real data. We would have to generate new bloom filters based on fake hostnames if we wanted fake hostnames here which didn't seem worth the effort, but LMK if you disagree.